airbyte-integrations/connectors/source-slab/manifest.yaml

```diff
@@ -0,0 +1,416 @@
+  entire organisation, from non-technical to tech-savvy. https://slab.com/
+
+
+  In order to use this source, you must first create an account and log in. 
+
+  The API uses a bearer token which can be obtained by navigating to Settings ->
+  Developer -> Admin Token.
+
+  You must be on a the business plan to obtain a token.
+
+  Slab uses Graphql API and this connector streams the `users`, `posts` and
+  `topics` endpoints. Find more information about the API here
+  https://studio.apollographql.com/public/Slab/variant/current/home 
+
+check:
+  type: CheckStream
+  stream_names:
+    - users
+
+definitions:
+  streams:
+    users:
+      type: DeclarativeStream
+      name: users
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: graphql
+          http_method: POST
+          request_body_json:
+            query: >-
+              query { organization { users { id name title email description
+              type deactivatedAt insertedAt updatedAt avatar { original thumb }
+              }} }
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - data
+              - organization
+              - users
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/users"
+    topics:
+      type: DeclarativeStream
+      name: topics
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: graphql
+          http_method: POST
+          request_body_json:
+            query: >-
+              query { organization { topics { id hierarchy description name
+              memberEditable insertedAt privacy updatedAt inheritParent parent {
+              id name } ancestors { id name } children { id name } banner {
+              original } } } }
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - data
+              - organization
+              - topics
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/topics"
+    posts:
+      type: DeclarativeStream
+      name: posts
+      primary_key:
+        - post_id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: graphql
+          http_method: POST
+          request_body_json:
+            query: >-
+              query { search(query: "posts" ,first: 100,  types: [POST]) { edges
+              { node { ... on PostSearchResult { post { id title content
+              insertedAt publishedAt archivedAt owner { id name email } topics {
+              id name } } } } }  } }
```

Comment:
I don't believe it GraphQL connectors are supported by the builder today. With this limitation it wn't be possible to accept.

Reply:
Ah ss that so :( the endpoints are being streamed